inc/template-functions.php

```diff
@@ -143,6 +143,24 @@ function hale_sidebar_location( $sidebar ) {
 
 function hale_get_branding_class() {
 
+	$css_file_path = "./wp/wp-includes/css/MjB-test.css";
+	$css_file_exists = file_exists($css_file_path);
+	$headingColour = get_theme_mod( 'heading_colour', '#0b0c0c' );
+	$cookieButtonColour = get_theme_mod( 'cookie_button_colour', '#0b0c0c' );
+
+	if ( $css_file_exists ) {
+		$css_file = fopen($css_file_path, "w") or die("Unable to open file!");
+		if (!empty($headingColour) ) {
+			$css = "h1, h2 {color:$headingColour!important;}";
+			fwrite($css_file, $css);
+		}
```

Comment:
We can write the CSS file in sections like this, so any blank fields are ignored.

Reply:
I wondering if we can do this in a string replace sort of way. So for example have css template with variables. That would mean rather then loads of if statements that may grow you just have an array of variables that get passed in. Each variable can have a default. Just thinking how we can make the code smaller. But happy to discuss.